src/modules/campath/mod.rs

```diff
@@ -162,6 +176,17 @@ fn force_live(marker: MainThreadMarker) {
     IS_FORCED.set(marker, true);
 }
 
+fn rotate_round_z(point: glam::Vec3, rad: f32) -> glam::Vec3 {
+    // only change x and y
+    let orig_x = point.x;
+    let orig_y = point.y;
+
+    let x = orig_x * rad.cos() - orig_y * rad.sin();
```

Comment:
Could probably use https://docs.rs/glam/latest/glam/f32/struct.Mat2.html#method.from_angle and then multiply by it